metrics/all_kind.go

```diff
@@ -0,0 +1,99 @@
+package metrics
+
+import (
+	"net/http"
+	"time"
+)
+
+type All struct {
+	prometheus      *Prometheus
+	codaHale        *CodaHale
+	prommHandler    http.Handler
```

Comment:
nit: maybe we should call it `prometheusHandler` `promm` with two m's looks wrong :)

Reply:
done